probe/cmd/probe/main.go

```diff
@@ -16,10 +19,28 @@ func main() {
 
 	// Always log to stderr by default, required for glog.
 	if err := flag.Set("logtostderr", "true"); err != nil {
-		glog.Info("Unable to set logtostderr to true.")
+		glog.Info("unable to set logtostderr to true.")
 	}
 
-	c, err := cli.New()
+	config, err := config.GetConfig()
+	if err != nil {
+		glog.Fatal(err)
+	}
+
+	if metricsServer := metrics.NewMetricsServer(config.MetricsAddress); metricsServer != nil {
+		defer func(server *http.Server) {
+			if err := server.Close(); err != nil {
+				glog.Errorf("failed to close metrics server: %v", err)
+			}
+		}(metricsServer)
+		go func() {
+			if err := metricsServer.ListenAndServe(); err != nil && err != http.ErrServerClosed {
```

Comment:
nit: Personally favor the `error.Is` call more, but looking at other sources and places this one's also valid, so you don't have to change it.
```suggestion
			if err := metricsServer.ListenAndServe(); err != nil && !errors.Is(err, http.ErrServerClosed) {
```